bundles/org.openhab.binding.meater/src/main/java/org/openhab/binding/meater/internal/handler/MeaterHandler.java

```diff
@@ -0,0 +1,171 @@
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.meater.internal.handler;
+
+import static org.openhab.binding.meater.internal.MeaterBindingConstants.*;
+
+import java.time.ZonedDateTime;
+
+import javax.measure.quantity.Temperature;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.meater.internal.MeaterConfiguration;
+import org.openhab.binding.meater.internal.dto.MeaterProbeDTO.Cook;
+import org.openhab.binding.meater.internal.dto.MeaterProbeDTO.Device;
+import org.openhab.core.i18n.TimeZoneProvider;
+import org.openhab.core.library.types.DateTimeType;
+import org.openhab.core.library.types.QuantityType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.library.unit.SIUnits;
+import org.openhab.core.library.unit.Units;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.Channel;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.binding.BaseThingHandler;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.RefreshType;
+import org.openhab.core.types.State;
+import org.openhab.core.types.UnDefType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link MeaterHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Jan Gustafsson - Initial contribution
+ */
+@NonNullByDefault
+public class MeaterHandler extends BaseThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(MeaterHandler.class);
+
+    private MeaterConfiguration config;
+    private TimeZoneProvider timeZoneProvider;
+
+    public MeaterHandler(Thing thing, TimeZoneProvider timeZoneProvider) {
+        super(thing);
+        this.timeZoneProvider = timeZoneProvider;
+        config = getConfigAs(MeaterConfiguration.class);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        logger.debug("Command received: {}", command);
+        if (CHANNEL_STATUS.equals(channelUID.getId()) || command instanceof RefreshType) {
+            update();
+        }
+    }
+
+    @Override
+    public void initialize() {
+        updateStatus(ThingStatus.UNKNOWN);
+
+        scheduler.execute(() -> {
+            update();
+        });
+    }
+
+    public void update() {
+        Device meaterProbe = getMeaterProbe();
+        if (meaterProbe != null) {
+            update(meaterProbe);
+        } else {
+            logger.debug("Meater probe is offline!");
```

Comment:
This logging is not needed as the framework will already log the `updateStatus()` below. Also, without thing id, the use is somewhat limited. Example for Meater Block:
```
2022-10-02 23:41:38.693 [DEBUG] [eater.internal.handler.MeaterHandler] - Meater probe is offline!
2022-10-02 23:41:38.695 [DEBUG] [eater.internal.handler.MeaterHandler] - Meater probe is offline!
2022-10-02 23:41:38.698 [DEBUG] [eater.internal.handler.MeaterHandler] - Meater probe is offline!
2022-10-02 23:41:38.700 [DEBUG] [eater.internal.handler.MeaterHandler] - Meater probe is offline!
```
(they are all offline because the Block is turned off)

Reply:
Removed logging.